tests/fixtures/module-unification-addon/tests/dummy/config/targets.js

```diff
@@ -1,18 +1,8 @@
-'use strict';
-
-const browsers = [
-  'last 1 Chrome versions',
-  'last 1 Firefox versions',
-  'last 1 Safari versions'
-];
-
-const isCI = !!process.env.CI;
-const isProduction = process.env.EMBER_ENV === 'production';
-
-if (isCI || isProduction) {
-  browsers.push('ie 11');
-}
-
 module.exports = {
-  browsers
+  browsers: [
```

Comment:
Looking at this diff I've realized that the module-unification-app blueprint is outdate compared with the regular `app` blueprint. We need to come up with a system that reuses as much as possible from the standard blueprint and only swaps the indispensable files, so the chances of this happening again is lower.

Reply:
> module-unification-app blueprint is outdate compared with the regular app blueprint

https://github.com/ember-cli/ember-cli/pull/7595 aims to address this by re-using the same `config/targets.js` fixture for both `app` and `module-unification-app` blueprints testing. 

Also I've been working on reducing duplication between these blueprints(https://github.com/ember-cli/ember-cli/pull/7500) but it was blocked due to bad testing story(I believe).